### ISSUE TITLE
Writing the Seed Makes the Land Very Rough

### ISSUE THREAD

**Issue by IAmSneak** [jaskarth/overworld-two#4]
![2020-08-22_18 51 27](https://user-images.githubusercontent.com/66091019/90967131-d6f15400-e4a8-11ea-9341-dde761381f50.png)
On the same seed the terrain is much less rough than when it is written in the seed box. The picture is of the world with the seed put in and both are on the world type "Overworld Two".

**Comment by Gegy** [comment]
Hmm, this is a really interesting issue! Thanks for the report :)

**Comment by jupiterio** [comment]
Hello, I had this issue too. Took the time to take a look at the level.dat of a normal Overworld Two world and compared it to one where I typed the same seed :)

The first thing I noticed is that the generator type of the overworld and nether is set to `minecraft:noise` and not `overworld_two:overworld_two`. Changing it to the correct value fixes the roughness but yields different terrain and biomes since it seems like a few other parameters are different as well.
